bsst/bsst_test.go

```diff
@@ -33,12 +35,13 @@ var _ Source = (*testSource)(nil)
 
 func TestBSSTCreate(t *testing.T) {
 
-	n := int64(13421280)
+	n := int64(1024)
```

Comment:
This is meant to be this large number because it is above the threshold where multiple levels will be created
```suggestion
	n := int64(13421280)
```

Reply:
On my machine this takes about a minute to run with n = `13421280`. Hopefully it won't take much longer than couple of minutes on the CI 🤞